src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,55 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "movie_sessions")
+public class MovieSession {
```

Comment:
you need to override 'toString' for this class, since you use it in exception messages

and be aware of lazy relations while overriding it